apps/aecore/src/aec_object_serialization.erl

```diff
@@ -78,6 +78,7 @@ tag(channel_slash_tx) -> 55;
 tag(channel_settle_tx) -> 56;
 tag(channel_offchain_tx) -> 57;
 tag(channel) -> 58;
+tag(trees_poi) -> 60;
 tag(block) -> 100.
```

Comment:
I understand the block is not part of consensus, so correctly not documented in [protocol](https://github.com/aeternity/protocol/blob/fb7059ee91617718ce8c8356c18685db13ca64d6/serializations.md#table-of-object-tags). May I suggest marking it with a comment in this file (e.g. `tag(block) -> 100 %% Not part of consensus.`)?